apis/docdb/v1alpha1/zz_db_cluster.go

```diff
@@ -0,0 +1,273 @@
+	// Specifies whether this cluster can be deleted. If DeletionProtection is enabled,
+	// the cluster cannot be deleted unless it is modified and DeletionProtection
+	// is disabled. DeletionProtection protects clusters from being accidentally
+	// deleted.
+	DeletionProtection *bool `json:"deletionProtection,omitempty"`
+	// A list of log types that need to be enabled for exporting to Amazon CloudWatch
+	// Logs. You can enable audit logs or profiler logs. For more information, see
+	// Auditing Amazon DocumentDB Events (https://docs.aws.amazon.com/documentdb/latest/developerguide/event-auditing.html)
+	// and Profiling Amazon DocumentDB Operations (https://docs.aws.amazon.com/documentdb/latest/developerguide/profiling.html).
+	EnableCloudwatchLogsExports []*string `json:"enableCloudwatchLogsExports,omitempty"`
+	// The name of the database engine to be used for this cluster.
+	//
+	// Valid values: docdb
+	// +kubebuilder:validation:Required
+	Engine *string `json:"engine"`
+	// The version number of the database engine to use. The --engine-version will
+	// default to the latest major engine version. For production workloads, we
+	// recommend explicitly declaring this parameter with the intended major engine
+	// version.
+	EngineVersion *string `json:"engineVersion,omitempty"`
+	// The AWS KMS key identifier for an encrypted cluster.
+	//
+	// The AWS KMS key identifier is the Amazon Resource Name (ARN) for the AWS
+	// KMS encryption key. If you are creating a cluster using the same AWS account
+	// that owns the AWS KMS encryption key that is used to encrypt the new cluster,
+	// you can use the AWS KMS key alias instead of the ARN for the AWS KMS encryption
+	// key.
+	//
+	// If an encryption key is not specified in KmsKeyId:
+	//
+	//    * If ReplicationSourceIdentifier identifies an encrypted source, then
+	//    Amazon DocumentDB uses the encryption key that is used to encrypt the
+	//    source. Otherwise, Amazon DocumentDB uses your default encryption key.
+	//
+	//    * If the StorageEncrypted parameter is true and ReplicationSourceIdentifier
+	//    is not specified, Amazon DocumentDB uses your default encryption key.
+	//
+	// AWS KMS creates the default encryption key for your AWS account. Your AWS
+	// account has a different default encryption key for each AWS Region.
+	//
+	// If you create a replica of an encrypted cluster in another AWS Region, you
+	// must set KmsKeyId to a KMS key ID that is valid in the destination AWS Region.
+	// This key is used to encrypt the replica in that AWS Region.
+	KMSKeyID *string `json:"kmsKeyID,omitempty"`
+	// The password for the master database user. This password can contain any
+	// printable ASCII character except forward slash (/), double quote ("), or
+	// the "at" symbol (@).
+	//
+	// Constraints: Must contain from 8 to 100 characters.
+	// +kubebuilder:validation:Required
+	MasterUserPassword *string `json:"masterUserPassword"`
```

Comment:
We cannot include sensitive information field on the CR, see this [section](https://github.com/crossplane/crossplane/blob/master/design/one-pager-managed-resource-api-design.md#sensitive-input-fields) for how to handle this case. [rds.DBCluster](https://github.com/crossplane/provider-aws/blob/5a380ad/pkg/controller/rds/dbcluster/setup.go#L84) is a good example on how to do that.

Reply:
Added an referencer for `MasterUserPassword` similar to `rds.DBCluster`.